docs/eventing/broker/broker-event-delivery.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Event delivery"
+weight: 03
+type: "docs"
+showlandingtoc: "false"
+---
+
+You can configure how events are delivered for each broker by adding a `delivery` spec, as shown in the following example:
+
+```yaml
+apiVersion: eventing.knative.dev/v1
+kind: Broker
+metadata:
+  name: with-dead-letter-sink
+spec:
+  delivery:
+    deadLetterSink:
+      ref:
+        apiVersion: serving.knative.dev/v1
+        kind: Service
+        name: example-sink
+    backoffDelay: <duration>
+    backoffPolicy: <policy-type>
+    retry: <integer>
+```
+
+Where
+
+- The `deadLetterSink` spec contains configuration settings to enable using a dead letter sink. This tells the subscription what happens to events that cannot be delivered to the subscriber. When this is configured, events that fail to be delivered are sent to the dead letter sink destination. The destination can be a Knative service or a URI. In the example, the destination is a `Service` object, or Knative service, named `example-sink`.
+- The `backoffDelay` delivery parameter specifies the time delay before an event delivery retry is attempted after a failure. The duration of the `backoffDelay` parameter is specified using the ISO 8601 format. For example, `PT1S` specifies a 1 second delay.
+- The `backoffPolicy` delivery parameter can be used to specify the retry back off policy. The policy can be specified as either `linear` or `exponential`. When using the `linear` back off policy, the back off delay is the time interval specified between retries. When using the `exponential` back off policy, the back off delay is equal to `backoffDelay*2^<numberOfRetries>`.
```

Comment:
There's some confusion about whether linear means "constant delay" or "linearly increasing delay". Here it's described as constant delay but IIRC the actual intent is linearly increasing, i.e. "the back off delay increases by the given interval for each retry."

For exponential, "the back off delay increases by some multiplier of the given interval for each retry."

It's probably best to keep the specifics of how backoff is calculated vague, because different implementations will have different definitions of what exponential backoff means.